docs/3.0/deploy/infrastructure-examples/serverless.mdx

```diff
@@ -267,26 +267,26 @@ run this command for your particular cloud provider:
 
     ```bash
-    ╭───────────────────────────────────────────────────────────────────────────────────────────╮
-    │ Provisioning infrastructure for your work pool my-aci-work-pool will require:                     │
-    │                                                                                           │
-    │     Updates in subscription Azure subscription 1                                          │
-    │                                                                                           │
-    │         - Create a resource group in location eastus                                      │
-    │         - Create an app registration in Azure AD prefect-aci-push-pool-app                │
-    │         - Create/use a service principal for app registration                             │
-    │         - Generate a secret for app registration                                          │
-    │         - Create an Azure Container Registry with prefix prefect                          │
-    │         - Create an identity prefect-acr-identity to allow access to the created registry │
-    │         - Assign Contributor role to service account                                      │
-    │         - Create an ACR registry for image hosting                                        │
-    │         - Create an identity for Azure Container Instance to allow access to the registry │
-    │                                                                                           │
-    │     Updates in Prefect workspace                                                          │
-    │                                                                                           │
-    │         - Create Azure Container Instance credentials block aci-push-pool-credentials     │
-    │                                                                                           │
-    ╰───────────────────────────────────────────────────────────────────────────────────────────╯
+    _____________________________________________________________________________________________
+    | Provisioning infrastructure for your work pool my-aci-work-pool will require:             |
+    |                                                                                           |
+    |     Updates in subscription Azure subscription 1                                          |
+    |                                                                                           |
+    |         - Create a resource group in location eastus                                      |
+    |         - Create an app registration in Azure AD prefect-aci-push-pool-app                |
+    |         - Create/use a service principal for app registration                             |
+    |         - Generate a secret for app registration                                          |
+    |         - Create an Azure Container Registry with prefix prefect                           |
```

Comment:
```suggestion
    |         - Create an Azure Container Registry with prefix prefect                          |
```

Reply:
Weirdly there was one spot that renders as a straight line, although it is off by one in the file.